GameServer/Logic/GameObjects/Unit.cs

```diff
@@ -603,37 +603,54 @@ public virtual void refreshWaypoints()
         public int ClassifyTarget(Unit target)
         {
             /*
-Under normal circumstances, a minion痴 behavior is simple. Minions follow their attack route until they reach an enemy to engage.
-Every few seconds, they will scan the area around them for the highest priority target. When a minion receives a call for help
-from an ally, it will evaluate its current target in relation to the target designated by the call. It will switch its attack
-to the new target if and only if the new target is of a higher priority than their current target. Minions prioritize targets
-in the following order:
-
-    1. An enemy champion designated by a call for help from an allied champion. (Enemy champion attacking an Allied champion)
-    2. An enemy minion designated by a call for help from an allied champion. (Enemy minion attacking an Allied champion)
-    3. An enemy minion designated by a call for help from an allied minion. (Enemy minion attacking an Allied minion)
-    4. An enemy turret designated by a call for help from an allied minion. (Enemy turret attacking an Allied minion)
-    5. An enemy champion designated by a call for help from an allied minion. (Enemy champion attacking an Allied minion)
-    6. The closest enemy minion.
-    7. The closest enemy champion.
-*/
+            Under normal circumstances, a minion behavior is simple. Minions follow their attack route until they reach an enemy to engage.
```

Comment:
This whole section could very well just be a docstring.

See here: https://msdn.microsoft.com/en-us/library/aa288481(v=vs.71).aspx